config.json

```diff
@@ -66,6 +66,7 @@
     "wordy",
     "allergies",
     "poker",
+    "all-your-base",
```

Comment:
Sure. I think the confusion comes from this post https://www.sitepoint.com/exorcise-your-newbie-demons-by-contributing-to-exercism/. In the "Updating the configuration" section it asks to "stick it anywhere in the second half". I think that document this would be great.